gen-web/src/index.js

```diff
@@ -33,13 +33,16 @@
   }
 
   const inputs = {
+    registrationCode: document.querySelector('#registration-code'),
     email: document.querySelector('#email'),
     password: document.querySelector('#password'),
     passwordCheck: document.querySelector('#password-check'),
   }
 
   const inlineVariables = {
     contentContainer: document.querySelector('#content-container'),
+    registrationCodePlaceholder: document.querySelector('#registration-code-placeholder'),
```

Comment:
This doesn't seem to be apart of the designs currently.  Let's remove it.

Reply:
done